exercises/luhn/canonical-data.json

```diff
@@ -123,7 +115,23 @@
       "expected": true
     },
     {
-      "description": "strings with non-digits is invalid",
+      "comments": [
+        "Convert non-digits to their ascii value and then offset them by 48 sometimes accidently return the correct result.",
```

Comment:
"accidentally return the correct result" might be easy to misinterpret, or require careful thinking before a reader can determine whether the sentence is true. Can I suggest "accidentally declare an invalid string to be valid" so that requires less brainpower? my feeble mind was having a hard time figuring things out